src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -51,42 +49,48 @@ function LtiConfigForm({
             }}
           />
         </p>
-        <Form.Group controlId="consumerKey" isInvalid={errors.consumerKey}>
-          <Form.Label>{intl.formatMessage(messages.consumerKey)}</Form.Label>
+        <Form.Group controlId="consumerKey" isInvalid={errors.consumerKey} className="mb-4">
           <Form.Control
+            className="text-gray-500"
+            controlClassName={errors.consumerKey && 'border-danger-300'}
+            floatingLabel={intl.formatMessage(messages.consumerKey)}
             onChange={handleChange}
             onBlur={handleBlur}
             value={values.consumerKey}
           />
           {errors.consumerKey && (
-          <Form.Control.Feedback type="invalid">
-            {errors.consumerKey}
+          <Form.Control.Feedback type="invalid" hasIcon={false} className="text-brand-500">
```

Comment:
Do we need to override the text color here? The override does not match the figma design. Also isn't the default color already red?

Reply:
Default colour is red but not same as figma. I used figma described class name but for current paragon version it is different. we do not need to override text colour unless we want 

> > > @awais-ansari The error message text-color does not seem to be red. Can we make it red?
> > 
> > 
> > I used text-brand-500 which is red shade as used in figma. Colours are little bit different for current paragon version. I checked **netlify** paragon version 14.7 have some colour as figma.
> 
> What is the default color of the error message?

default color is #C32D3A but figma has #D23228

Reply:
We need to be careful here and use our judgment.  This is an error message, and so should probably use the 'danger' styles.  If Figma says otherwise, we should question that.  I disagree that we should change error messages to be a brand color instead of the danger color.  I'd like us to bring this back to Jon to be revisited.

Reply:
I totally agree with your point. We need to bring this back to Jon to be revisited.